### PR TITLE
KGCL prototype library and webapp

### PR DESCRIPTION
**Pull request by ckindermann**
This is just a bare-bones Flask application that shows how I instantiate classes in the KGCL data model for testing purposes.

https://kgcl.ontodev.com/

### CODE REVIEW

File: parser/parser.py [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+def parse_statement(input): 
+
+    tree = kgcl_parser.parse(input)
+
+    id = "test_id_" + str(next(id_gen))
+
+    command = tree.data
+    #print("Command: " + command)
+
+    if(command == "rename"):
```
Comment (cmungall) [inline parser/parser.py:38]: minor: can we write this as more idiomatic python. @jamesaoverton do you have a preferred linter - flake8, black, ..?

File: parser/parser.py [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+        label_token = next(get_tokens(label))
+
+
+        replacement_id = tree.find_data('replacement')
+
+        replacement_id_list = list(tree.find_data('replacement'))
+        if(replacement_id_list):
+            t = next(replacement_id)
+            replacement_token = next(get_tokens(t))
+            return python.kgcl.NodeObsoletion(id=id, about_node=label_token, has_direct_replacement=replacement_token)
```
Comment (cmungall) [inline parser/parser.py:66]: minor: we should fix imports, should be either

```
from kgcl import NodeObsoletion, ...

...

return NodeObsoletion(...
```

or perhaps


```
import kgcl
...

return kgcl.NodeObsoletion(...
```

File: parser/parser.py (hunk 1 above)
Comment (jamesaoverton) [reply parser/parser.py:38]: flake8 and black are good for different things, so Knocean usually uses both. @ckindermann please follow https://github.com/knocean/practises/tree/master/python#style and ask me if you have any questions.

Comment (ckindermann) [reply parser/parser.py:38]: @jamesaoverton will do!

**Pull request merged**